extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -964,6 +964,29 @@ private static String getVideoInfoUrl(final String id, final String sts) {
                                     + deobfuscateSignature(cipher.get("s"));
                         }
 
+                        int bitrate = formatData.getInt("bitrate");
+                        int width = formatData.getInt("width");
+                        int height = formatData.getInt("height");
+                        JsonObject initRange = formatData.getObject("initRange");
+                        JsonObject indexRange = formatData.getObject("indexRange");
+                        int initStart = Integer.parseInt(initRange.getString("start", "-1"));
+                        int initEnd = Integer.parseInt(initRange.getString("end", "-1"));
+                        int indexStart = Integer.parseInt(indexRange.getString("start", "-1"));
+                        int indexEnd = Integer.parseInt(indexRange.getString("end", "-1"));
+                        int fps = formatData.getInt("fps");
+                        String mimeType = formatData.getString("mimeType", EMPTY_STRING);
+                        String codec = mimeType.contains("codecs") ? mimeType.split("\"")[1] : EMPTY_STRING;
+
+                        itagItem.bitrate = bitrate;
```

Comment:
Why did you create a bitrate variable which is never used more than once, instead of using `itagItem.bitrate = formatData.getInt("bitrate")`?